ivy/functional/backends/numpy/linear_algebra.py

```diff
@@ -388,10 +388,19 @@ def vector_norm(
     elif ord == -inf:
         return np.min(abs_x, axis=axis, keepdims=keepdims, out=out)
     else:
-        return (
+        # There is a rounding error whenever the input is a 0-dim
+        # The solution at the moment is to convert the 0-dim array to 1-dim
+        # and then convert it back to 0-dim in case of keepdims=True
+        abs_x = np.array(np.abs(x), ndmin=1)
```

Comment:
I think we could just do a `np.abs(x).expand_dims` instead based on the dimensions, `np.array` creates a new copy of the array ig
Happy to know what you think, thanks @fabriziocacicia 😄

Reply:
Good point @vedpatwardhan 
Let me try and test it with `expand_dims`

Reply:
Just done!